go.mod

```diff
@@ -5,25 +5,30 @@ go 1.18
 replace github.com/urfave/cli => github.com/keep-network/cli v1.20.0
 
 require (
-	github.com/BurntSushi/toml v0.3.1
 	github.com/btcsuite/btcd v0.22.1
 	github.com/btcsuite/btcd/btcec/v2 v2.2.0
 	github.com/ethereum/go-ethereum v1.10.19
 	github.com/gogo/protobuf v1.3.2
 	github.com/google/gofuzz v1.1.1-0.20200604201612-c04b05f3adfa
+	github.com/hashicorp/go-multierror v1.1.1
 	github.com/ipfs/go-datastore v0.5.1
 	github.com/ipfs/go-ipfs-config v0.16.0
 	github.com/ipfs/go-log v1.0.5
 	github.com/jbenet/goprocess v0.1.4
-	github.com/keep-network/keep-common v1.7.1-0.20220726115135-ff5f20e7eb1b
+	github.com/keep-network/keep-common v1.7.1-0.20220728224545-2fde28814493
```

Comment:
```
github.com/keep-network/keep-common v1.7.1-0.20220729135717-12e41453bfba
```

Reply:
One more we need my friend: https://github.com/keep-network/keep-common/pull/96

Reply:
Merged. Side note, we should consider moving the `Config` struct out of `keep-common` to `keep-core`. But not in this PR.